sriovnet.go

```diff
@@ -433,6 +433,24 @@ func GetVfIndexByPciAddress(vfPciAddress string) (int, error) {
 	return -1, fmt.Errorf("vf index for %s not found", vfPciAddress)
 }
 
+// gets the PF index that's associated with a VF PCI address (e.g '0000:03:00.4')
+func GetPfIndexByVfPciAddress(vfPciAddress string) (int, error) {
```

Comment:
Overall Looks good.

can you add a UT for it ?

Reply:
Done. PTAL

Reply:
i dont see the UT, did you push the latest changes ?

Reply:
Pushed it.